templates/user/notifications/new_featured_conference.html.twig

```diff
@@ -0,0 +1,2 @@
+New featured conference ! See it here : <a href="{{ path('conferences_show', { slug: notification.conference.slug }) }}" class="text-primary">See Conference</a>.
```

Comment:
```suggestion
New featured conference! See it here : <a href="{{ path('conferences_show', { slug: notification.conference.slug }) }}" class="text-primary">See Conference</a>.
```

I think you can rework all message. There are too "raw".
And in this case it'll render as:

`New featured conference ! See it here : See Conference.`

**Maybe** something like `A new featured conference has been added` (and the link on all the text).